lib/app.js

```diff
@@ -24,21 +24,89 @@ app.use('/api', ensureAuth);
 // and now every request that has a token in the Authorization header will have a `req.userId` property for us to see who's talking
 app.get('/api/test', (req, res) => {
   res.json({
-    message: `in this proctected route, we get the user's id like so: ${req.userId}`
+    message: `in this protected route, we get the user's id like so: ${req.dessertsId}`
   });
 });
 
-app.get('/animals', async(req, res) => {
+app.post('/desserts', async (req, res) => {
```

Comment:
So close! This should have been your `GET /desserts route` which returns a list of all of your desserts.